engine/src/main/java/nl/inl/blacklab/search/DocUtil.java

```diff
@@ -13,59 +13,21 @@
 import nl.inl.blacklab.exceptions.BlackLabRuntimeException;
 import nl.inl.blacklab.search.indexmetadata.AnnotatedField;
 import nl.inl.blacklab.search.indexmetadata.Field;
-import nl.inl.blacklab.search.results.Hit;
+import nl.inl.blacklab.search.results.EphemeralHit;
 import nl.inl.blacklab.search.results.Hits;
 import nl.inl.blacklab.search.results.ResultsStats;
 import nl.inl.util.XmlHighlighter;
 import nl.inl.util.XmlHighlighter.HitCharSpan;
 
-public class DocImpl implements Doc {
-    
-    /** Index the document is in */
-    private BlackLabIndex index;
-    
-    /** Lucene document id */
-    private int id;
-    
-    /** Lucene document (if cached) */
-    private Document document;
-    
-    public DocImpl(BlackLabIndex index, int id) {
-        this.index = index;
-        this.id = id;
-    }
-
-    @Override
-    public BlackLabIndex index() {
-        return index;
-    }
+/**
+ * Several utility methods related to documents and highlighting.
+ */
+public class DocUtil {
```

Comment:
Potential improvement here, a lot this methods here have `id` as a parameter. It is unclear what `id` is referring to. Until reading the body of the method. It would be great to either rename this do `docId` or a simple java doc explaning the argument

Reply:
Agreed, and did both.